common/tools/dev-tool/src/commands/run/testNodeTSInput.ts

```diff
@@ -21,23 +21,17 @@ export const commandInfo = makeCommandInfo(
 );
 
 export default leafCommand(commandInfo, async (options) => {
-  const isModuleProj = await isModuleProject();
   const reporterArgs =
     "--reporter ../../../common/tools/mocha-multi-reporter.js --reporter-option output=test-results.xml";
-  const defaultMochaArgs = `${
-    isModuleProj ? "--loader=ts-node/esm " : ""
-  }-r ts-node/register ${reporterArgs} --full-trace`;
+  const defaultMochaArgs = `${reporterArgs} --full-trace`;
   const updatedArgs = options["--"]?.map((opt) =>
     opt.includes("**") && !opt.startsWith("'") && !opt.startsWith('"') ? `"${opt}"` : opt,
   );
   const mochaArgs = updatedArgs?.length
     ? updatedArgs.join(" ")
     : '--timeout 1200000 --exclude "test/**/browser/*.spec.ts" "test/**/*.spec.ts"';
   const command = {
-    command: isModuleProj
-      ? `mocha ${defaultMochaArgs} ${mochaArgs}`
-      : // eslint-disable-next-line no-useless-escape
-        `cross-env TS_NODE_COMPILER_OPTIONS="{\\\"module\\\":\\\"commonjs\\\"}" mocha ${defaultMochaArgs} ${mochaArgs}`,
+    command: `mocha --require tsx ${defaultMochaArgs} ${mochaArgs}`,
```

Comment:
I think we may have missed it in #28808 - @mpodwysocki is this change ok to go in?

Reply:
Correction: #28801

Reply:
This work should be reverted and use the TSX specific files and not these base ones here.

Reply:
Yup yup, can do - just need some clarification on https://github.com/Azure/azure-sdk-for-js/pull/28826/files#r1520494618 before I make code changes